### ISSUE TITLE
Data Science with Python : Data Analysis using Pandas

### ISSUE THREAD

**Issue by prathimacode-hub** [girlscript/winter-of-contributing#767]
Welcome to 'DSWP' Team, good to see you here

This issue will helps readers in acquiring all the knowledge to learn about getting involved with Data Analysis using Pandas. Tutorial to Data Analysis using Pandas and how it's applied using sample code.

To get assigned to this issue, add your serial numbers mentioned in the spreadsheet of "Data Science with Python", the approach one would follow and choice you prefer (Documentation, Audio, Video). You can go with all three or any number of options you're interested to work on.

This issue is created for beginners and who aren't assigned to any issues yet.

If you had referred any resources, add them up in "DS Resources". Similarly if you had used datasets, include them in "DS Datasets".

Domain : Data Analysis and Visualization

Mentors Assigned : Arya Patel / Shivang Dubey

Points to Note :

The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.

"Issue Title" and "PR Title should be the same. Include issue number along with it.

Changes should be made inside the Datascience_With_Python/ directory & Datascience_With_Python branch.

Follow Contributing Guidelines & Code of Conduct before start Contributing.

This issue is only for 'GWOC' contributors of 'DSWP' domain.

All the best. Enjoy your open source journey ahead. 😎

**Comment by AdityaStark7** [comment]
Hey.
I want to work on this issue
Batch: 03
serial no. 374

**Comment by AdityaStark7** [comment]
Aditya Kumar
Type: Document
Serial no. 374

**Comment by prathimacode-hub** [comment]
Hey @AdityaStark7, this issue is done on contributor request as per thread mentioned in discord. if you are up for audio/video, you can state your interest.

If not stay awaited, there are many more issues upcoming to be created.

**Comment by vanshika230** [comment]
I would like to work on this.
Vanshika Mishra - 830
Batch - 19
Type: Documentation
Approach:
1. What is Pandas library ?
2.  Creating dataframes
3.  Loc and iloc
4.  Data Analysis on test.csv as example
5.  Reading CSV files
6.  Reading URL to CSV
7. Reading HTML to CSV
8. Reading EXCEL to CSV

**Comment by prathimacode-hub** [comment]
Issue assigned to @vanshika230 for documentation

**Comment by Sakshi21112003** [comment]
name-sakshi singh
serial no-409
type -documentation

**Comment by prathimacode-hub** [comment]
Documentation is assigned, if you're up for audio/video let me know. @Sakshi21112003

**Comment by Sakshi21112003** [comment]
@prathimacode-hub  i am interested

**Comment by prathimacode-hub** [comment]
> @prathimacode-hub i am interested

For? Audio/video or Both? Also add the approach you would follow along with your choice. @Sakshi21112003

**Comment by Shashankkrj** [comment]
@prathimacode-hub Can you help me regarding video part ? Actually,I want to know that we have to only record the code or information using screen recorder or we have to do something else in the video part of this issue.

**Comment by deepthi1107** [comment]
Hello, I am Deepthi M with a serial number:172. I would like to do documentation on this issue.
Approach: Dealing with dataset and write all the related code with a detailed comments.

**Comment by prathimacode-hub** [comment]
> @prathimacode-hub Can you help me regarding video part ? Actually,I want to know that we have to only record the code or information using screen recorder or we have to do something else in the video part of this issue.
> 
> I think that it is same as documentation part and only thing that we have to do is record that document in the form of video and submit it.

I will update with details in discord.

**Comment by prathimacode-hub** [comment]
> Hello, I am Deepthi M with a serial number:172. I would like to do documentation on this issue.
> Approach: Dealing with dataset and write all the related code with a detailed comments.

I guess you had been assigned to another issue. @deepthi1107

**Comment by deepthi1107** [comment]
Yes, but I have written the document, I have a doubt on how to do PR. I will be doing it soonly @prathimacode-hub

**Comment by m0hit-kumar** [comment]
i want to contribute

**Comment by Aaksha-sharma** [comment]
Aaksha Sharma
DSWP batch 18
I want to contribute through documentation

**Comment by Sakshi21112003** [comment]
@prathimacode-hub 

> > @prathimacode-hub i am interested
> 
> For? Audio/video or Both? Also add the approach you would follow along with your choice. @Sakshi21112003

>video  through ppt ,sketch animation topics abot library how to visualise data

**Comment by anamhayat** [comment]
name- anam hayat
serial no - 936
batch no -21
type- documentation

**Comment by prathimacode-hub** [comment]
> @prathimacode-hub
> 
> > > @prathimacode-hub i am interested
> > 
> > 
> > For? Audio/video or Both? Also add the approach you would follow along with your choice. @Sakshi21112003
> 
> > video  through ppt ,sketch animation topics abot library how to visualise data

Ok, but tell the approach - what's the outline of the topic you are going to cover @Sakshi21112003

**Comment by Sakshi21112003** [comment]
> > @prathimacode-hub
> > > > @prathimacode-hub i am interested
> > > 
> > > 
> > > For? Audio/video or Both? Also add the approach you would follow along with your choice. @Sakshi21112003
> > 
> > 
> > > video  through ppt ,sketch animation topics abot library how to visualise data
> 
> Ok, but tell the approach - what's the outline of the topic you are going to cover @Sakshi21112003

> approach - why pandas,loading and reading datas , saving data in csv/exel,sorting and filtering

**Comment by shripaddhopate** [comment]
I would be glad to work on this issue.
Name : Shripad Dhopate
Batch : 21
Type : Documentation , Video
Serial No: 930

**Comment by prathimacode-hub** [comment]
@Sakshi21112003 is assigned for video contribution

@shripaddhopate, you can chose audio
